src/app/components/AllowanceMenu/index.tsx

```diff
@@ -87,6 +87,7 @@ function AllowanceMenu({ allowance, onEdit, onDelete }: Props) {
             <TextField
               id="budget"
               label="Budget"
+              autoFocus
```

Comment:
Added these to inputs in modals to improve UX.  
Turns out it doesn't seem to work. WIll double check and create an issue if needed.